Scripts/script-DBotPredictTextLabel.yml

```diff
@@ -0,0 +1,125 @@
+commonfields:
+  id: DBotPredictTextLabel
+  version: -1
+name: DBotPredictTextLabel
+script: |+
+  import sys, base64, re
+  import demisto_ml
+
+  THRESHOLD = int(demisto.args().get('warning_threshold', 50))
+  HASH_WORDS = (demisto.args()['hashData'] == 'yes')
+  HASH_SEED = 5381
+
+  def get_input_text_prediction(model, input_text):
+      number_of_tokens = len(input_text.split(" "))
+      about_threshold = True
```

Comment:
@erezh31 what do we used `about_threshold` for..?

Reply:
removed